### PR TITLE
[stdlib] Minor tensorflow cleanup

### PR DESCRIPTION
**Pull request by anayini**
A few minor code cleanup and lint fixes for the new Tensorflow
stdlib code.

### CODE REVIEW

**Comment by anayini** [comment]
@rxwei - Reopened here - sorry for the delay - does this look right?

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 1]
```diff
@@ -133,11 +132,10 @@ public enum _RuntimeConfig {
 }
 
 private func configureRuntimeFromEnvironment() {
-  if let value = getenv("SWIFT_TENSORFLOW_ENABLE_DEBUG_LOGGING") {
-    if String(cString: value).lowercased() == "true" {
+  if let value = getenv("SWIFT_TENSORFLOW_ENABLE_DEBUG_LOGGING"),
+    String(cString: value).lowercased() == "true" {
```
Comment (compnerd) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]: Why the chaining here?  None of the other cases do that.  The rest LGTM

Comment (anayini) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: I figured the chaining was better than an extra nested `if`.  Did the same below on `160`.  Or maybe not understanding you correctly @compnerd

Comment (compnerd) [reply stdlib/public/TensorFlow/CompilerRuntime.swift]: Oh, I see, the folded bit had confused me.  This seems reasonable.

**Comment by compnerd** [comment]
@swift-ci please test and merge

**Comment by anayini** [comment]
@compnerd - Is the above just a flake?

**Comment by compnerd** [comment]
@anayini - could be, but there are merge conflicts now. Can you rebase your patch please?

**Comment by rxwei** [comment]
We've just set up CI. Would you like to resolve the conflicts and test it?

**Comment by anayini** [comment]
@compnerd @rxwei -  will do after work today

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Review by rxwei — APPROVED** [review]

**Comment by rxwei** [comment]
@swift-ci please test tensorflow

**Pull request merged**